### PR TITLE
Zetachain Network Updation

### PR DESCRIPTION
**Pull request by Kali-Decoder**
This PR aims to refresh the network details of ZetaChain for both its mainnet and testnet environments. The update is likely intended to ensure Wagmi, a tool or platform that interacts with blockchain networks, has accurate information about ZetaChain. With this updated data, users of Wagmi would be able to seamlessly interact with ZetaChain through Wagmi's functionalities.
<br/>
Screenshot 2024-06-04 at 3 58 01 PM

---

## PR-Codex overview
The focus of this PR is to update chain definitions for Zetachain Athens Testnet and Zetachain Mainnet.

### Detailed summary
- Updated chain name and native currency for Zetachain Athens Testnet
- Added testnet flag for Zetachain Athens Testnet
- Updated RPC URL and block explorer URL for Zetachain Athens Testnet
- Added contracts information for Zetachain Athens Testnet
- Updated chain name for Zetachain Mainnet
- Updated native currency for Zetachain Mainnet
- Updated RPC URL and block explorer URL for Zetachain Mainnet
- Added contracts information for Zetachain Mainnet

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: src/chains/definitions/zetachainAthensTestnet.ts [hunk 1]
```diff
@@ -2,22 +2,29 @@ import { defineChain } from '../../utils/chain/defineChain.js'
 
 export const zetachainAthensTestnet = /*#__PURE__*/ defineChain({
   id: 7001,
-  name: 'ZetaChain Athens Testnet',
+  name: "Zetachain Athens 3 Testnet",
+  testnet:true,
+  iconUrl: "https://s2.coinmarketcap.com/static/img/coins/64x64/21259.png",
   nativeCurrency: {
+    name: "ZetaChain Athens 3 Testnet",
```
Comment (fadeev) [inline src/chains/definitions/zetachainAthensTestnet.ts:9]:
```suggestion
    name: "ZetaChain Testnet",
```

ZetaChain only has one testnet, so I think it's easier if we just call it "Testnet" without the "Athens" codename.

File: src/chains/definitions/zetachainAthensTestnet.ts [hunk 2]
```diff
@@ -2,22 +2,29 @@ import { defineChain } from '../../utils/chain/defineChain.js'
 
 export const zetachainAthensTestnet = /*#__PURE__*/ defineChain({
   id: 7001,
-  name: 'ZetaChain Athens Testnet',
+  name: "Zetachain Athens 3 Testnet",
```
Comment (fadeev) [inline src/chains/definitions/zetachainAthensTestnet.ts:5]:
```suggestion
  name: "ZetaChain Testnet",
```

File: src/chains/definitions/zetachain.ts [hunk 3]
```diff
@@ -1,23 +1,20 @@
...
   id: 7000,
-  name: 'ZetaChain',
-  nativeCurrency: {
-    decimals: 18,
-    name: 'Zeta',
-    symbol: 'ZETA',
-  },
+  name: "Zetachain Mainnet",
+  iconUrl: "https://s2.coinmarketcap.com/static/img/coins/64x64/21259.png",
+  nativeCurrency: { name: "Zetachain", symbol: "ZETA", decimals: 18 },
```
Comment (fadeev) [inline src/chains/definitions/zetachain.ts:7]:
```suggestion
  nativeCurrency: { name: "ZetaChain", symbol: "ZETA", decimals: 18 },
```

File: src/chains/definitions/zetachain.ts [hunk 4]
```diff
@@ -1,23 +1,20 @@
...
-export const zetachain = /*#__PURE__*/ defineChain({
+export const zetachain = /*#__PURE__*/defineChain({
   id: 7000,
-  name: 'ZetaChain',
-  nativeCurrency: {
-    decimals: 18,
-    name: 'Zeta',
-    symbol: 'ZETA',
-  },
+  name: "Zetachain Mainnet",
```
Comment (fadeev) [inline src/chains/definitions/zetachain.ts:5]:
```suggestion
  name: "ZetaChain Mainnet",
```

File: src/chains/definitions/zetachain.ts [hunk 5]
```diff
@@ -1,23 +1,20 @@
...
-    default: {
-      http: ['https://zetachain-evm.blockpi.network/v1/rpc/public'],
-    },
+    default: { http: ["https://zetachain-evm.blockpi.network/v1/rpc/public"] },
   },
   blockExplorers: {
-    default: {
-      name: 'ZetaScan',
-      url: 'https://explorer.zetachain.com',
+    default: { name: "zetachain", url: "https://explorer.zetachain.com/" },
```
Comment (fadeev) [inline src/chains/definitions/zetachain.ts:12]:
```suggestion
    default: { name: "ZetaScan", url: "https://explorer.zetachain.com/" },
```

**Review by Kali-Decoder — COMMENTED** [review]
Done Changes

File: src/chains/definitions/zetachain.ts [hunk 6]
```diff
@@ -1,23 +1,20 @@
...
+export const zetachain = /*#__PURE__*/defineChain({
   id: 7000,
-  name: 'ZetaChain',
-  nativeCurrency: {
-    decimals: 18,
-    name: 'Zeta',
-    symbol: 'ZETA',
-  },
+  name: "ZetaChain Mainnet",
+  iconUrl: "https://s2.coinmarketcap.com/static/img/coins/64x64/21259.png",
```
Comment (jxom) [inline src/chains/definitions/zetachain.ts:6]: Please remove

File: src/chains/definitions/zetachainAthensTestnet.ts [hunk 7]
```diff
@@ -2,22 +2,29 @@ import { defineChain } from '../../utils/chain/defineChain.js'
...
+  testnet:true,
+  iconUrl: "https://s2.coinmarketcap.com/static/img/coins/64x64/21259.png",
```
Comment (jxom) [inline src/chains/definitions/zetachainAthensTestnet.ts:7]: Please remove

**Comment by Kali-Decoder** [comment]
Done Above Changes @jxom

**Review by jxom — CHANGES_REQUESTED** [review]
Please fix the formatting.

`pnpm lint`

**Comment by jxom** [comment]
Please re-open once above comment is addressed.

**Pull request closed without merging**